src/ORM/BehaviorRegistry.php

```diff
@@ -99,6 +98,19 @@ protected function _resolveClassName($class)
         return $result;
     }
 
+    /**
+     * Resolve a behavior classname.
+     *
+     * Part of the template method for Cake\Core\ObjectRegistry::load()
+     *
+     * @param string $class Partial classname to resolve.
+     * @return string|false Either the correct classname or false.
+     */
+    protected function _resolveClassName($class)
```

Comment:
how we can find this when porting to 4.x for removing

Reply:
@saeideng This won't be removed in 4.0 since it's an inherited abstract method `ObjectRegistry::_resolveClassName()`, implemented by all child classes.